tests/test_policies.py

```diff
@@ -198,9 +198,12 @@ def test_memorise(self, trained_policy, default_domain):
         trackers = train_trackers(default_domain)
         trained_policy.train(trackers, default_domain)
 
+        test_trackers = \
+            [t for t in trackers if not
+             hasattr(t, 'is_augmented') or not t.is_augmented]
         (all_states, all_actions) = \
             trained_policy.featurizer.training_states_and_actions(
-                trackers, default_domain)
+                test_trackers, default_domain)
```

Comment:
could you please also add assert that for aug trackers `recalled` is array of zeros